application/helpers/expressions/em_manager_helper.php

```diff
@@ -8508,6 +8508,8 @@ public static function ProcessCurrentResponses()
                                             }
                                             $aFiles[$i]->filename = $sDestinationFileName;
                                         }
+                                        /* Sanitize size */
+                                        $aFiles[$i]->size = floatval($aFiles[$i]->size);
```

Comment:
Use `sanitize_float` from `sanitize_helper`?

Reply:
1. I really don't want to allow 1,2 : it must be 1.2, real float.
2. It must be float for round, not GMP number 

https://github.com/LimeSurvey/LimeSurvey/blob/b95bf50c0dd76788d906978de4f589fd86c75ef2/application/helpers/sanitize_helper.php#L341